### PR TITLE
[DataNormalizer] treat download-url as code_repository

### LINKED ISSUE #97: Capture download_url from setup.py
Package level flows fail if the link to GitHub repo is stored under `download_url` instead of `url`. An example could be httpie: https://github.com/jakubroztocil/httpie/blob/master/setup.py#L79

To solve this, we should propagate only GitHub urls in the `url` field.

### PR DESCRIPTION
**Pull request by jpopelka**
Fixes #97

### CODE REVIEW

File: f8a_worker/data_normalizer.py [hunk 1]
```diff
@@ -242,20 +242,25 @@ def _split_keywords(keywords, separator=','):
         return keywords
 
     def _handle_python(self, data):
-        "Handle Python package (setup.py) analysis data"
+        """ setup.py """
```
Comment (pkajaba) [inline f8a_worker/data_normalizer.py]: Just a little nitpicking - this is not related to this PR that much, so it would be nice to do it as a separate commit.

**Comment by pkajaba** [comment]
Thanks, LGTM

File: f8a_worker/data_normalizer.py (hunk 1 above)
Comment (jpopelka) [inline f8a_worker/data_normalizer.py]: ok, next time :-)
